lib/AccessSystem/TelegramBot.pm

```diff
@@ -499,6 +497,101 @@ sub inductions ($self, $message) {
     }
 }
 
+=head2 make_inductor
+
+=cut
+
+sub make_inductor ($self, $message, $args = undef) {
+    return unless $self->authorize($message);
+
+    my $reply = ref($message) =~ /Callback/ ? 'answer' : 'reply';
+    my ($tool, $person, $p_status, $person_or_keyb, $t_status, $tool_or_keyb);
+
+    # Requestor is a valid member of the makerspace (paid up)
+    my $member = $self->member($message);
+    return if !$member;
+
+    # Something's very wrong if we have no Door .. 
+    my $door = $self->db->resultset('Tool')->find({ name => 'The Door' });
+    return if !$door;
+
+    ## Must be an admin, or existing inductor
+    my $allowed = $member->allowed->find({ tool_id => $door->id });
+    return if !$allowed;
+    # if (!$allowed->is_admin) {
+    #     return $message->$reply("You're not allowed to do that");
+    # }
+
+    if (!$args && $message->text =~ m{^/make_inductor\s([\w\s]+)\son\s([\w\s\d]+)$}) {
```

Comment:
Will \w capture names that include accents?

Reply:
Actually a general thing I've been wondering about re this system

Reply:
Good question it should ... just tested with some random polish string : "/inductions małego psa" (thats an L with a strike through in the middle) and it does look that up just fine.. however it won't pick up double-barrelled (aka "with -") surnames, or irish ones with ' in.. so should probably expand it a bit.. 

Sadly regexp::common (a library of common regexes) doesn't seem to have one for "names"..